frontend/src/metabase/nav/components/ProfileLink.jsx

```diff
@@ -62,18 +62,18 @@ export default class ProfileLink extends Component {
         });
 
         return (
-            <OnClickOutsideWrapper handleDismissal={this.closeDropdown}>
-                <div className={dropDownClasses}>
-                    <a data-metabase-event={"Navbar;Profile Dropdown;Toggle"} className="NavDropdown-button NavItem flex align-center p2 transition-background" onClick={this.toggleDropdown}>
-                        <div className="NavDropdown-button-layer">
-                            <div className="flex align-center">
-                                <UserAvatar user={user} style={{backgroundColor: 'transparent'}}/>
-                                <Icon name="chevrondown" className="Dropdown-chevron ml1" size={8} />
-                            </div>
+            <div className={dropDownClasses}>
+                <a data-metabase-event={"Navbar;Profile Dropdown;Toggle"} className="NavDropdown-button NavItem flex align-center p2 transition-background" onClick={this.toggleDropdown}>
+                    <div className="NavDropdown-button-layer">
+                        <div className="flex align-center">
+                            <UserAvatar user={user} style={{backgroundColor: 'transparent'}}/>
+                            <Icon name="chevrondown" className="Dropdown-chevron ml1" size={8} />
                         </div>
-                    </a>
+                    </div>
+                </a>
 
-                    { dropdownOpen ?
+                { dropdownOpen ?
```

Comment:
It's basically a style preference, but I think there are a couple minor advantages to using ternary:

1. The order of operations of ternary vs boolean operators sometimes lets you leave off parenthesis, e.x. `a || b && <div />` could incorrectly evaluate to `a`, so you have to do `(a || b) && <div />`, but  `a || b ? <div /> : null` is fine. Of course the lack of parenthesis are offset by the addition of `: null`, but at least the logic won't be wrong.
2. If you decide you want to add an `else` you don't need to change the top part.
3. Consistency (except we're not consistent)